x/dsmr/block.go

```diff
@@ -116,6 +158,15 @@ type Block struct {
 	blkBytes []byte
 }
 
+func NewBlock(parentID ids.ID, height uint64, timestamp int64, chunkCerts []*ChunkCertificate) Block {
```

Comment:
nit: I don't think we need this diff?

Reply:
I know that you don't like intuitive constructors... I'll roll back this change. ( we use it in only a single place ).